File: 3 - Linguagem C++/2 - Estrutura-de-repeticao-enquanto-ler-valor-x-dar-soma-pares-consecutivos.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main()
{
   int x, somapar;

   cout << "Digite um numero inteiro: " << endl;
   cin >> x;

   somapar = 0;

   while (x != 0) {
       if (x % 2 != 0){
          x = x + 1;
       }

       somapar = 5 * x  + 20;
       cout << "SOMA = " << somapar << endl;

       cout << "Digite um numero inteiro: " << endl;
       cin >> x;
   }

    return 0;
}

File: 4 - JAVA/2 - Estrutura-de-repeticao-enquanto-ler-valor-x-dar-soma-pares-consecutivos.java
import java.util.Locale;
import java.util.Scanner;

public class Main {
	
	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int x, soma = 0;
		
		System.out.print("Digite um numero inteiro: ");
		x = sc.nextInt();
		
		while (x != 0) {
			if (x % 2 != 0) {
				x = x + 1;
			}
			
			soma = 5 * x + 20;
			System.out.println("SOMA = " + soma);
			
			System.out.println("Digite um numero inteiro: ");
			x = sc.nextInt();			
		}
		
		sc.close();

	}
}